asttoc/src/cppmm_ast_write_cmake.cpp

```diff
@@ -113,6 +120,84 @@ void cmake(const char* project_name, const Root& root, size_t starting_point,
         }
         out.print(")\n");
         out.print("target_link_libraries (${{LIBNAME}} ${{{}}})\n", lib_var);
+        out.print("target_link_libraries (${{LIBNAME}}-shared ${{{}}})\n",
```

Comment:
MInor, but It might be worth using the `PRIVATE` option when calling `target_link_libraries` as well.